### DISCUSSION TITLE
Manipulating the window — can I get the underlying handle to the window? (eframe/eframe template)

### DISCUSSION THREAD

**Discussion by reivilibre** [emilk/egui#906] [Q&A]
Hi (and thanks for a very very pleasant UI crate!),

I'd like to do some manipulation of the window (for example: hiding/showing it, repositioning it to a fixed place on-screen, and so on).

I suspect it would be best to use `winit` to mess around with this stuff. Is there a way that I can get a handle to the `winit` Window (or equivalent)?

I'm using `eframe`, based on the `eframe_template`. (Of course, I understand these shenanigans won't work on web and I'm happy to accept that.)

Thank you!

**Comment by emilk** [comment]
No, not right now. You can try using `egui_glow` directly instead.
